### PR TITLE
feat: add feedback link to UI + update feed count

### LINKED ISSUE #265: Add feedback link to UI + update feed count
[Add feedback button (same style as the Web UI for the GTFS Schedule Validator) to the footer](https://www.figma.com/file/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?type=design&node-id=989-908&mode=dev)

[Update feed count from 1800 on the homepage to 2000](https://www.figma.com/file/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?type=design&node-id=656-435&mode=dev) 

Please add me as a reviewer to this one so I can see how to make static changes easily to the UI in the future and I won't bog you folks down with minor content changes.

### PR DESCRIPTION
**Pull request by cka-y**
**Summary:**
Closes #265 

**Expected behavior:** 
✅ Added feedback lint to `Footer.tsx`
✅ Updated feed count from 1800+ to 2000+

image

Please make sure these boxes are checked before submitting your pull request - thanks!
- [x] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [x] Add or update any needed documentation to the repo
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: web-app/src/app/components/Footer.tsx [hunk 1]
```diff
@@ -14,6 +14,20 @@ const Footer: React.FC = () => {
 
   return (
     <footer className='footer'>
+      <a
+        href={'https://share.mobilitydata.org/validator-feedback'}
```
Comment (emmambd) [inline web-app/src/app/components/Footer.tsx:18]: Link should be updated: https://share.mobilitydata.org/mobility-database-feedback

**Review by emmambd — COMMENTED** [review]
@cka-y Link needs to be fixed, otherwise looks good!

**Review by qcdyx — APPROVED** [review]
LGTM

File: web-app/src/app/components/Footer.tsx (hunk 1 above)
Comment (cka-y) [reply web-app/src/app/components/Footer.tsx]: done!

**Review by emmambd — APPROVED** [review]
@cka-y LGTM!

**Pull request merged**